plugins/chunter-resources/src/components/CommentsPresenter.svelte

```diff
@@ -26,12 +26,14 @@
 </script>
 
 {#if value && value > 0}
+  <!-- svelte-ignore a11y-click-events-have-key-events -->
   <div
     use:tooltip={{
       label: chunter.string.Comments,
       component: CommentPopup,
       props: { objectId: object._id }
     }}
+    on:click|preventDefault|stopPropagation={() => {}}
```

Comment:
why?

Reply:
To allow click on comments, if inside space selector.